### ISSUE TITLE
[2.1.17] iPhone 6 / 10.2 - Application is crashing while exporting lomotif after user navigates to "In progress" tab from "Following" tab and then tries to export the lomotif from "In progress" tab.

### ISSUE THREAD

**Issue by Pankaj-Testershub** [Lomotif/lomotif-ios-issues#1550]
Frequency: Every time

Action:
1: Launch the app and login with a valid credential.
2: Swipe right to reveal "Following" tab and select one own lomotif.
3: Tap on own username and navigate to "Profile>>In progress" screen.
4: Load one lomotif and export the lomotif, skip sharing and navigate to "Following" tab(Observe user will be navigated to own "Profile" screen as mentioned in step 3).
5: Swipe left to reveal "Home" screen and tap on "User" icon, the user will be navigated to "Profile>>In progress" screen.
6: Load one lomotif(Not the same lomotif which is loaded in step 4), the user will be navigated to "Create" screen.
7: Tap on the export option and tap on "Make Lomotif" button, exporting process will be started.
8: Check if the application is crashing.

Expected result: Application should not crash and user should be able to export/upload the lomotif successfully.

Actual result: Application is crashing while exporting lomotif after the user navigates to "In progress" tab from "Following" tab and then tries to export the lomotif.

Refer following video link for more details:
https://drive.google.com/open?id=0B0-oVt0Gfy3iU1duMEtVMzNDbGc

**Comment by Aishwarya-Testershub** [comment]
@xiuie I can reproduce this issue for build [2.1.17] on iPhone 7 / 10.2

**Comment by xiuie** [comment]
Was a similar bug like this logged before?

**Comment by Pankaj-Testershub** [comment]
@xiuie No, it is not similar to defect logged before, as here user is able to swipe left to reveal "Home" screen(Mentioned in step 5), then navigate to "Profile>>In progress" tab and then tries to export the lomotif.

**Comment by Pankaj-Testershub** [comment]
@xiuie No, it is not similar to defect logged before, as here user is able to swipe left reveal "Home"(Mentioned in step 5), then navigate to "Profile>>In progress" tab and then tries to export the lomotif.
